types/syslogpriority.pp

```diff
@@ -0,0 +1 @@
+type Puppetboard::Syslogpriority = Enum['debug', 'info', 'notice', 'warning', 'err', 'crit', 'alert', 'emerg']
```

Comment:
should we depend on puppet-tea here? https://github.com/voxpupuli/puppet-tea/blob/master/types/syslogpriority.pp

Reply:
I thought about that, but paused for two reasons... one is just adding another dependency for a single type that's unlikely to change; the other is this:
https://github.com/voxpupuli/puppet-tea/issues/19
I think the comments there make a lot of sense.

Reply:
We should check that all our types go into stdlib, since that finally is puppet4 only...